corehq/apps/app_manager/static/app_manager/js/details/column.js

```diff
@@ -75,6 +76,21 @@ hqDefine("app_manager/js/details/column", function () {
         self.fontSize = ko.observable(self.original.font_size || 'medium');
         self.fontSizeOptions = ['small', 'medium', 'large'];
 
+        self.showBorderOptions = ['no border', 'show border'];
```

Comment:
nit: could this be simplifies with a checkbox since its a binary option or are we anticipating more options in the future?